superset/migrations/versions/30bb17c0dc76_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 30bb17c0dc76
+Revises: f231d82b9b26
+Create Date: 2018-04-08 07:34:12.149910
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '30bb17c0dc76'
+down_revision = 'f231d82b9b26'
+
+from datetime import date
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    with op.batch_alter_table('logs') as batch_op:
+        batch_op.drop_column('dt')
```

Comment:
@john-bodley let's avoid backward incompatible db migrations in the future as they require scheduling downtime to upgrade the app for people who use rolling deploys. This took our production down for a moment.

Reply:
@mistercrunch just to clarify what should have the logic been here? I'm simply asking to learn.